packages/kit/test/build-errors/package.json

```diff
@@ -3,7 +3,8 @@
 	"private": true,
 	"version": "0.0.0-next.0",
 	"scripts": {
-		"test": "uvu . \"(spec\\.js|test[\\\\/]index\\.js)\""
+		"test": "uvu . \"(spec\\.js|test[\\\\/]index\\.js)\"",
+		"test:cross-browser": "uvu . \"(spec\\.js|test[\\\\/]index\\.js)\""
```

Comment:
perhaps we should rename the script from `cross-browser` to `cross-platform` since there's no browser involved here. (we don't necessarily have to rename the folder used by the browser tests, but just the script key in the `package.json`)